src/controllers/userControllers.ts

```diff
@@ -1,19 +1,35 @@
+//import { NextFunction } from "express";
+import { User, validateUser } from "../models/User";
+import bcrypt from "bcrypt";
+import _ from "lodash";
+
 //const config = require('../config/Config');
-const _ = require("lodash");
-const { User, validateUser } = require("../models/User");
-const bcrypt = require("bcrypt");
 
-const createError = (msg, code = 403) => {
-  const err = new Error(msg);
-  //console.log(message);
-  err.code = code;
-  //console.log(err);
+class customError extends Error {
```

Comment:
Abstract the `customError` into a single module that can be used by all controllers